test/models/collapsible_map_test.rb

```diff
@@ -394,6 +394,18 @@ def test_mapping_one_location
     assert_obj_arrays_equal([loc], mapset.underlying_locations)
   end
 
+  # CollapsibleCollection should ignore large locations and observations with
+  # vague locations like "California", when mapping multiple locations.
+  # (:california is currently one of the locations for a "Boletus edulis" obs.)
+  def test_mapping_collections_ignores_vague_locations
```

Comment:
Question: I don't understand Mappable well enough (i.e., my understanding is little to none) to answer the following:
**Does this test prove anything?**
In particular, if this test were part of the current main branch, would it pass?

Reply:
Good question. It would not pass. `locations(:california)` is one of the locations of a "Boletus edulis" observation in the fixtures, so it's included in the set of observations passed to `CollapsibleCollection` and also returned by it.